README.md

```diff
@@ -17,7 +17,7 @@ Before you get started:
 - Install Go `1.11` or later
 - Latest version of `dep`
 - Kubernetes Cluster `1.12` or later (e.g. [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/))
-- [Configure kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) with version `1.12` or later
+- [Install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) with version `1.12` or later
```

Comment:
`Configure` gives the impression that there is a special configuration for kubectl which following the link will expose.   That is not the case... the link is to install... if it said that I would not have followed the link as I already have it.